core/spec.md

```diff
@@ -138,6 +138,8 @@ For easy reference, the JSON serialization of a Registry adheres to this form:
 
   "model": {                            # Only if inlined
     "labels": { "STRING": "STRING" * }, ?
+    "modelversion": "STRING", ?         # Version of the model
+    "compatiblewith": "URI", ?          # Statement of compatibility with model spec
```

Comment:
Minor thing, but move these 2 to be before "labels", I think from an "importance" perspective, if they're specified I think they're more important than "labels", and I try to put the most important stuff first.

I tend to view "labels" as a bag for extensions... and so I try to keep them near the end of the list of attributes.